pkgs/by-name/cy/cy/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, nix-update-script, go, }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cy";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "cfoust";
+    repo = "cy";
+    rev = "b385923fd350254df53df26e4462a2b860c1cb1c";
```

Comment:
is there a tag you can lock to instead? if you can please use that + the version of that tag for the version attr above